scripts/gen_FCHL19_rep.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python3
```

Comment:
If we're going to be adhering to PEP8

https://www.python.org/dev/peps/pep-0008/

I believe we'd require a module(script)-level docstring here.

Reply:
Would be interesting to see what Pylint has to say